src/components/00-materials/icons-raw/close.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="currentColor" d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
\ No newline at end of file
+<svg width="24" height="24" viewBox="0 0 21 21" fill="none" xmlns="http://www.w3.org/2000/svg"><line x1="1.41421" y1="1" x2="8" y2="7.58579" stroke="#00008F" stroke-width="2" stroke-linecap="round"/><line x1="13.4142" y1="13" x2="20" y2="19.5858" stroke="#00008F" stroke-width="2" stroke-linecap="round"/><line x1="1" y1="19.5858" x2="19.5858" y2="1" stroke="#00008F" stroke-width="2" stroke-linecap="round"/></svg>
```

Comment:
What's this change about?

Reply:
That was not a change. I have moved the material "close.svg" to the subfolder. And moved the internal-icon "close.svg" to the icons-raw folder. So it seems to be a change, but it is another icon.